ansible/roles/products/common/resources/templates/scripts/close-everything.j2

```diff
@@ -24,15 +24,15 @@ COMMON_ROSNODE_KILL_COMMAND="rosnode list | grep /bag_rotate | xargs -I{} rosnod
   echo 10
   echo "# Stopping Shadow Robot processes in the server container"
   docker exec $CONTAINER_NAME bash -c "rosnode kill $SERVER_RECORD; $COMMON_ROSNODE_KILL_COMMAND" || true &
-  sleep 2
+  sleep 0.2
   echo 20
-  sleep 2
+  sleep 0.2
   echo 30
-  sleep 2
+  sleep 0.2
   echo 40
-  sleep 2
+  sleep 0.2
   echo 50
-  sleep 2
+  sleep 0.2
```

Comment:
Are we sure the nodes will have died by the time we stop the container?
And, does it matter?
How did we choose the times in the first place?

Reply:
Pretty sure those times were chosen arbitrarily. However, just in case, this (ef058b3fac31d26fbed7b3626e02ef73978f752c) should now function the same as the previous version, but allow early continuation if the `rosnode kill` commands finish before the ten seconds worth of sleeps

Reply:
Nice!